### PR TITLE
Update Minecraft wiki links to new domain

### PR DESCRIPTION
**Pull request by Spongecade**
The Minecraft Fandom wiki has been forked to a new domain: minecraft.wiki. Learn more here: https://minecraft.wiki/w/Minecraft_Wiki:Moving_from_Fandom. This PR updates all URLs accordingly.

### CODE REVIEW

File: projects/core/src/main/resources/data/computercraft/lua/rom/help/commandsapi.txt [hunk 1]
```diff
@@ -12,4 +12,4 @@ commands.give( "dan200", "minecraft:diamond", 64 )
 This works with any command. Use "commands.async" instead of "commands" to execute asynchronously.
 
 The commands API is only available on Command Computers.
-Visit http://minecraft.gamepedia.com/Commands for documentation on all commands.
+Visit http://minecraft.wiki/w/Commands for documentation on all commands.
```
Comment (SquidDev) [inline projects/core/src/main/resources/data/computercraft/lua/rom/help/commandsapi.txt]:
```suggestion
Visit https://minecraft.wiki/w/Commands for documentation on all commands.
```

File: projects/core/src/main/resources/data/computercraft/lua/rom/help/keys.txt [hunk 2]
```diff
@@ -6,4 +6,4 @@ if sEvent == "key" and nKey == keys.enter then
    -- Do something
 end
 
-See http://www.minecraftwiki.net/wiki/Key_codes, or the source code, for a complete reference.
+See http://www.minecraft.wiki/w/Key_codes, or the source code, for a complete reference.
```
Comment (SquidDev) [inline projects/core/src/main/resources/data/computercraft/lua/rom/help/keys.txt]:
```suggestion
See https://www.minecraft.wiki/w/Key_codes, or the source code, for a complete reference.
```

**Review by SquidDev — CHANGES_REQUESTED** [review]
Thank you! Cannot believe I did this to the original CC wiki, and not my own documentation >_>.

Can you update the two links I've mentioned below to be `https://`, but otherwise looks good to me!

**Comment by Spongecade** [comment]
All good!

**Review by SquidDev — APPROVED** [review]
Fantastic, thank you!

**Pull request merged**